environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentModificationService.java

```diff
@@ -74,11 +78,20 @@ public class EnvironmentModificationService {
 
     private final AzureParametersRepository azureParametersRepository;
 
-    public EnvironmentModificationService(EnvironmentDtoConverter environmentDtoConverter, EnvironmentService environmentService,
-            CredentialService credentialService, NetworkService networkService, AuthenticationDtoConverter authenticationDtoConverter,
-            ParametersService parametersService, EnvironmentFlowValidatorService environmentFlowValidatorService,
-            EnvironmentResourceService environmentResourceService, EnvironmentEncryptionService environmentEncryptionService,
-            AzureParametersRepository azureParametersRepository) {
+    private final DnsV1Endpoint dnsV1Endpoint;
```

Comment:
it is not used (yet?)

Reply:
I do not get it. :smile: From `editFreeIPA()`:
```
            dnsV1Endpoint.addDnsZoneForSubnetIds(addDnsZoneForSubnetIdsRequest);
```